app/src/main/java/com/example/tasky/login/components/ButtonWithLoader.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.tasky.login.components
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.tasky.R
+import com.example.tasky.ui.theme.Black
+
+@Composable
+fun ButtonWithLoader(
+    modifier: Modifier = Modifier,
+    isLoading: Boolean = false,
```

Comment:
ButtonWithLoader, is a button you will always use with a isLoading parameter

So it doesnt make much sense to have a default value to it

If its always false, then you would usr a regular button and not a Button with loader